services/app-api/utils/validation/schemaMap.ts

```diff
@@ -122,7 +122,7 @@ export const endDate = (startDateField: string) =>
 
 // DROPDOWN
 export const dropdown = () =>
-  string().typeError(error.INVALID_GENERIC).required(error.REQUIRED_GENERIC);
+  object({ label: text(), value: text() }).required(error.REQUIRED_GENERIC);
```

Comment:
Devnotes: updated dropdown schema to be similar to how we handle choice fields. You'll see in this change in the ui-src/ schemas as well